### PR TITLE
Add comments for agreements_subscription_agreement_org_ext.sql

### PR DESCRIPTION
**Pull request by arthur-aguilera**
Related to #632. Adding field definitions as comments

```
All queries:
- [ ] Release notes added or updated in CHANGES.md
- [ ] Query runs without errors
- [ ] Query output is correct
- [ ] Query logic is clear and well documented
- [ ] Query is readable and properly indented with spaces (not tabs)
- [ ] Table and column names are in all-lowercase
- [ ] Quotation marks are used only where necessary

Report queries:
- [ ] Query has complete user documentation
    - [ ] Purpose of report
    - [ ] Sample output
    - [ ] Query instructions

Derived tables:
- [ ] Query begins with user documentation in comment lines
- [ ] File name is listed in `runlist.txt` after dependencies
- [ ] All columns have indexes
- [ ] Table is vacuumed and analyzed
```

### CODE REVIEW

**Comment by amzoss** [comment]
Hi @arthur-aguilera! Can you please also add information to CHANGES.md about the changes you're making? I'll wait to review until I see that edit come through.

**Comment by stdombek** [comment]
Closes #624 #632

**Comment by stdombek** [comment]
- invoice_adjustments_ext.sql
- invoice_adjustments_in_addition_to.sql
- po_lines_cost.sql

These updates don't have a GitHub issue and just add comments.

**Review by stdombek — APPROVED** [review]

**Comment by stdombek** [comment]
Please only enter the changes in CHANGES.md.

**Comment by arthur-aguilera** [comment]
> Hi @arthur-aguilera! Can you please also add information to CHANGES.md about the changes you're making? I'll wait to review until I see that edit come through.

Thank you, @amzoss and @stdombek, for the reminder! I committed notes to the CHANGES.md file. Stefan, when you say semicolons, are you talking about the semicolons within the sql files for each table?

**Comment by stdombek** [comment]
> > Hi @arthur-aguilera! Can you please also add information to CHANGES.md about the changes you're making? I'll wait to review until I see that edit come through.
> 
> Thank you, @amzoss and @stdombek, for the reminder! I committed notes to the CHANGES.md file. Stefan, when you say semicolons, are you talking about the semicolons within the sql files for each table?

These are semicolons inside the comments. In the description of the API, many developers use semicolons, among other things. The semicolons are not a problem for running in LDP and DBeaver. I'm not sure if it works in all programs though. 
Example: invoice_adjustments_ext.sql > line 96 in this PR
It's a general issue that we should discuss in the DEV meeting. I had also seen the case in another PR.

**Review by stdombek — APPROVED** [review]

File: sql_metadb/derived_tables/invoice_adjustments_ext.sql [hunk 1]
```diff
@@ -81,4 +81,28 @@ CREATE INDEX ON invoice_adjustments_ext (ratio_of_inv_adj_per_invoice_line);
```
Comment (amzoss) [inline sql_metadb/derived_tables/invoice_adjustments_ext.sql:81]: For some reason, when I ran this query against folio_lotus, I got an error on the derived table `invoice_adjustments_in_addition_to` saying the relation does not exist (line 51). When I specify the `folio_derived` schema for that table, it works fine. I didn't have this trouble when running this query against folio_juniper, but I thought I would mention it.

**Review by amzoss — APPROVED** [review]
Overall these additions work fine. I did find a typo, though, and had one problem running a query that probably is just a transient error? I'm not sure, since that query is already running fine on our test environments.

Even with the typo, I am tempted to approve. That typo is probably straight from the developer API documentation, so it shouldn't really hurt anything if our database also has a typo.

All queries:
- [x] Release notes added or updated in CHANGES.md
- [?] Query runs without errors
- [x] Query output is correct
- [x] Query logic is clear and well documented
- [x] Query is readable and properly indented with spaces (not tabs)
- [x] Table and column names are in all-lowercase
- [x] Quotation marks are used only where necessary

Derived tables:
- [x] Query begins with user documentation in comment lines
- [x] File name is listed in `runlist.txt` after dependencies
- [x] All columns have indexes
- [x] Table is vacuumed and analyzed

File: sql_metadb/derived_tables/agreements_subscription_agreement_org_ext.sql [hunk 2]
```diff
@@ -39,4 +39,22 @@ CREATE INDEX ON agreements_subscription_agreement_org_ext (sao_note);
 
 CREATE INDEX ON agreements_subscription_agreement_org_ext (org_orgs_uuid);
 
+COMMENT ON COLUMN agreements_subscription_agreement_org_ext.sao_id IS 'UUID for subscription and organization pairing';
+
+COMMENT ON COLUMN agreements_subscription_agreement_org_ext.subscription_agreement_id IS 'UUId for the subscritpion and agreement pairing';
```
[the remark this answers is not in the document]
Comment (arthur-aguilera) [reply sql_metadb/derived_tables/agreements_subscription_agreement_org_ext.sql:44]: Hi @amzoss! Thank you for reviewing this! I fixed the typos and would like to resubmit it. If the PR was approved, am I too late to commit further changes?

**Comment by amzoss** [comment]
No, you should still be able to commit another change after approval, I
think

File: sql_metadb/derived_tables/finance_transaction_invoices.sql [hunk 3]
```diff
@@ -51,6 +51,7 @@ FROM
     LEFT JOIN folio_finance.fund AS tf ON ft.tofundid = tf.id
     LEFT JOIN folio_finance.budget AS fb ON ft.fromfundid = fb.fundid AND ft.fiscalyearid = fb.fiscalyearid
     LEFT JOIN folio_organizations. organizations AS oo ON jsonb_extract_path_text(ii.jsonb, 'vendorId')::uuid = oo.id
+    
 WHERE (jsonb_extract_path_text(ft.jsonb, 'transactionType') = 'Pending payment'
```
Comment (nassibnassar) [inline sql_metadb/derived_tables/finance_transaction_invoices.sql:55]: Be careful not to make accidental changes, in this case only a harmless blank line inserted into the query.

**Pull request merged**